src/redux/actions/task/task.js

```diff
@@ -0,0 +1,111 @@
+import {
+    ALL_TASKS,
+    ADD_TASK,
+    CHANGE_STATUS,
+    UPDATE_TASK,
+    GET_TASK,
+    DELETE_TASK,
```

Comment:
we need to create an action for 'Delete task' as well, so that deleting a task (called in the menu in the todolist) so that it is propagated also to the backend.